servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -3566,7 +3617,7 @@ String enforcedUserAuthorityFilter(final String roleUserAuthorityFilter, final S
     }
 
     boolean shouldValidateRoleMembers(final String userAuthorityFilter) {
-        return validateUserRoleMembers || validateServiceRoleMembers || userAuthorityFilter != null;
+        return validateGroupRoleMembers || validateUserRoleMembers || validateServiceRoleMembers || userAuthorityFilter != null;
```

Comment:
This method is always going to return true, so does it make sense to just remove it?

Reply:
I see all references are removed?

Reply:
eventually yes, it will be removed. I've removed in a couple of place already. once other functionality is added, it will slowly be removed

Reply:
ok, looks like it got cleaned up in all place, so I'll remove it in this PR as well.